src/phylum/init/cli.py

```diff
@@ -126,12 +191,9 @@ def version_check(version):
     if not version.startswith("v"):
         version = f"v{version}"
 
-    try:
-        # The release layout structure changed starting with v2.0.0 and support here is only for the new layout
-        if Version("v2.0.0") > Version(canonicalize_version(version)):
-            raise argparse.ArgumentTypeError("version must be at least v2.0.0")
-    except InvalidVersion as err:
-        raise argparse.ArgumentTypeError("an invalid version was provided") from err
+    supported_versions = supported_releases()
+    if version not in supported_versions:
+        raise argparse.ArgumentTypeError(f"version must be from a supported release: {', '.join(supported_versions)}")
```

Comment:
This format string breaks GitHub's syntax highlighting. And also breaks my brain.... An explicit call to `.format()` seems more zen.

```suggestion
        raise argparse.ArgumentTypeError("version must be from a supported release: {0}".format(', '.join(supported_versions)))
```

Reply:
Weird...single quotes inside of a double quoted string broke the highlighting? This could be more clean/clear...changes coming.